docs/email/components/images.html

```diff
@@ -90,6 +90,12 @@
             <img src="https://placehold.it/256" width="128" height="128" alt="alt_text" border="0" style="display: block;">
         </div>
     </div>
+    <div class="grid ai-start s-notice s-notice__info mb48">
+        <div class="grid--cell mr8">{% icon Alert %}</div>
+        <div class="grid--cell lh-lg">
+            <strong>Note:</strong> Every image that should <strong>scale down</strong> in small screens should have a <code class="stacks-code bg-powder-200">width: 100%; max-width: (desktop-width)px;</code> in it's <code class="stacks-code bg-powder-200">style=""</code> attribute. This allows images to scale down when their desktop width exceeds the width of its container.
```

Comment:
I don't quite understand this _(desktop-width)px_ part even having frontend background. Although, I'm not sure how to better explain it.

Reply:
Hmm, I see how that can be confusing, I'll clarify what I mean by (desktop-width)px.